cache.py

```diff
@@ -0,0 +1,92 @@
+import hashlib
+import inspect
+import os
+import pathlib
+from functools import wraps
+
+import appdirs
+import daiquiri
+from diskcache import Cache
```

Comment:
It's not obvious to me why you're importing the Cache symbol here rather than just ``diskcache``? We only use it once in the file, and then it looks like a locally defined class. I can sort-of see the reasoning for ``wraps``

Reply:
Yep, good point. Fixed!